pkg/cmd/roachtest/cluster.go

```diff
@@ -2365,7 +2365,7 @@ func (c *clusterImpl) RunE(ctx context.Context, options install.RunOptions, args
 	defer l.Close()
 
 	cmd := strings.Join(args, " ")
-	c.t.L().Printf("running cmd `%s` on nodes [%v]; details in %s.log", roachprod.TruncateString(cmd, 30), nodes, logFile)
+	l.Printf("running cmd `%s` on nodes [%v]; details in %s.log", roachprod.TruncateString(cmd, 30), nodes, logFile)
```

Comment:
I think this is slightly inaccurate as we want the main logger for the test/operation, not the specific separate log file for the command. For now I think we should just gate this behind `if c.t != nil {...}`.